tello Test/tello_web_server.py

```
@@ -799,6 +1550,283 @@ async def rotate_handler(request: web.Request) -> web.Response:
+            "messages": [{"role": "user", "content": message}],
+            "threadId": thread_id,
+            "resourceId": resource_id
+        }
+        
+        timeout = aiohttp.ClientTimeout(total=30)
+        async with aiohttp.ClientSession(timeout=timeout) as session:
+            async with session.post(
+                mastra_url,
+                json=payload,
+                headers={"Content-Type": "application/json"}
+            ) as resp:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)